### PR TITLE
simplify testing/specification setup

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

**Comment by dblock** [comment]
Personally, I find those gemfiles *super* annoying because they always get out of sync. Removing Jeweler is a good idea though.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,9 @@
 language: ruby
-
 rvm:
   - 1.9.3
   - 2.1.2
-  - jruby-19mode
```
Comment (dblock) [inline .travis.yml]: You no love JRuby and are you against the capitalist enterprise ruby too? :)

Comment (afeld) [inline .travis.yml]: Haha. Mostly impatient for builds.

**Comment by afeld** [comment]
> I find those gemfiles super annoying because they always get out of sync

Since they effectively inherit from the gemspec and just restrict the Mongoid version, they shouldn't need to change, aside from adding new ones for new major Mongoid versions. Seems like a non-issue.

**Comment by dblock** [comment]
Superseded with https://github.com/mongoid/mongoid-locker/pull/45.

**Pull request closed without merging**